src/Microsoft.AspNet.Mvc.Core/ModelBindingHelper.cs

```diff
@@ -530,5 +524,189 @@ public static object ConvertValuesToCollectionType<T>(Type modelType, IList<T> v
                 return null;
             }
         }
+
+        /// <summary>
+        /// Converts the provided <paramref name="value"/> to a value of <see cref="Type"/> <typeparamref name="T"/>
+        /// using the <see cref="CultureInfo.InvariantCulture"/>.
+        /// </summary>
+        /// <typeparam name="T">The <see cref="Type"/> for conversion.</typeparam>
+        /// <param name="value">The value to convert."/></param>
+        /// <returns>
+        /// The converted value or the default value of <typeparamref name="T"/> if the value could not be converted.
+        /// </returns>
+        public static T ConvertTo<T>(object value)
+        {
+            return ConvertTo<T>(value, culture: null);
+        }
+
+        /// <summary>
+        /// Converts the provided <paramref name="value"/> to a value of <see cref="Type"/> <typeparamref name="T"/>.
+        /// </summary>
+        /// <typeparam name="T">The <see cref="Type"/> for conversion.</typeparam>
+        /// <param name="value">The value to convert."/></param>
+        /// <param name="culture">The <see cref="CultureInfo"/> for conversion.</param>
+        /// <returns>
+        /// The converted value or the default value of <typeparamref name="T"/> if the value could not be converted.
+        /// </returns>
+        public static T ConvertTo<T>(object value, CultureInfo culture)
+        {
+            var converted = ConvertTo(value, typeof(T), culture);
+            return converted == null ? default(T) : (T)converted;
```

Comment:
Why not `return (T)converted;`? The extra complexity seems to duplicate the `Activator.CreateInstance(type)` case below.